install/kubernetes/cilium/values.yaml

```diff
@@ -33,7 +33,7 @@ global:
   registry: docker.io/cilium
 
   # tag is the container image tag to use
-  tag: latest
+  tag: 1.8.0-rc1
```

Comment:
on master, this has to be `latest` I believe. We are only doing non-master tags in release branches.

Reply:
yes, this was not intended - but it does need to reflect the proper version in the release branch which it presently does not